CHANGELOG.md

```diff
@@ -2,6 +2,12 @@ Change Log
 ==========
 
 
+2.1.0
+-----
+
+* Add the ability to retrieve all the vocabularies as JSON through /all.json and /all-verbose.json.
+
```

Comment:
Could also mention all.py and all-verbose.py now.

Reply:
true, I'll add those